src/Renderer/Shader/PointsFS.glsl

```diff
@@ -1,9 +1,15 @@
+#include <itowns/WebGL2_pars_fragment>
 #include <itowns/precision_qualifier>
 #include <logdepthbuf_pars_fragment>
 #if defined(USE_TEXTURES_PROJECTIVE)
 #include <itowns/projective_texturing_pars_fragment>
 #endif
 
+#ifdef GL_OES_standard_derivatives
+#extension GL_OES_standard_derivatives : enable
```

Comment:
We don't seem to use this extension, is this needed ?

Reply:
i remove it